nss-day-cohort-45/tabloidfullstack-giving-gillyflowers
# Am create comment

## Changes
* The picture size and post UI is handled #63 in the post detail
* User information is received server-side for security #68
*  Post Detail page now has a comment form that accessed by a show hide btn #20 

 **Front End**
      `PostDetail.js` and `CommentForm.js` work together to display the add new comments form

 **Back End**
`CommentController.cs` returns properly

## To Test
1) fetch and run and log in
2) Navigate to Post
3) Choose any post that you would like to add comments to
4) Click the add comment btn
5) Try to save a comment with a blank subject (you should not be able to. btn disabled)
5) Fill in the form
6) Save and notice the comment list show up
